src/app/components/Account/RuntimeAccountDetailsView.tsx

```diff
@@ -14,7 +14,7 @@ import { DashboardLink } from '../../pages/ParatimeDashboardPage/DashboardLink'
 import { AllTokenPrices } from '../../../coin-gecko/api'
```

Comment:
@donouwens check preview links above

one issue: missing "Name appointed by Oasis" indicator in some views

Reply:
Discussed: lets fix it in assets, not in code. We should have full control over the icons anyway.

Icons like https://assets.oasis.io/explorer-tokens/rose-illuminex.svg are a single asset, not two, so making it a circle would crop its border a bit.

Reply:
(I should make a script to generate these dual icon combinations)